scripts/ccf/acl/down.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../..")"
+
+acl-down() {
```

Comment:
shouldn't we call this acl-delete?

Reply:
the other are bring up containers, then up/down makes sense. But here you are creating a new ACL instance where create makes more sense.

Reply:
Why are containers any different from other services in terms of using this naming scheme? My understanding is up/down can refer to any cloud provisioning. 

Probably more importantly, having another name would require a bunch of code to handle that some CCF targets use `up.sh` and others will use `create.sh` so I don't think it's worth diverging even if create was more appropriate